lntest/fee_service.go

```diff
@@ -42,6 +43,9 @@ const (
 	// DefaultFeeRateSatPerKw specifies the default fee rate used in the
 	// tests.
 	DefaultFeeRateSatPerKw = 12500
+
+	// httpReadHeaderTimeout sets a timeout for the HTTP server.
+	httpReadHeaderTimeout = 5 * time.Second
```

Comment:
I think we should export the constant from the first commit and then use that here.

Reply:
Is the current change what you meant?